packages/core/src/types/oas3.ts

```diff
@@ -508,22 +508,36 @@ const SecurityScheme: NodeType = {
     openIdConnectUrl: { type: 'string' },
   },
   required(value) {
-    if (!value?.type) {
-      return ['type'];
+    switch (value?.type) {
+      case 'apiKey':
+        return ['type', 'name', 'in'];
+      case 'http':
+        return ['type', 'scheme'];
+      case 'oauth2':
```

Comment:
add support oauth2 flow object like in `oas2.ts`
https://spec.openapis.org/oas/v3.0.0#oauth-flows-object